chsdi/templates/htmlpopup/kataster_belasteter_standorte_militaer.mako

```diff
@@ -0,0 +1,20 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+    lang = lang if lang in ('fr', 'it') else 'de'
+    standorttyp = 'standorttyp_%s' % lang
+    status_altlv = 'status_altlv_%s' % lang
+    untersuchungsmassnahmen = 'untersuchungsmassnahmen_%s' % lang
+    str_output = c['attributes'][untersuchungsmassnahmen].replace('##', '<br />')
+%>
+
+    <tr><td class="cell-left">${_('ch.vbs.kataster-belasteter-standorte-militaer.katasternummer')}</td>                        <td>${c['attributes']['katasternummer'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.vbs.kataster-belasteter-standorte-militaer.standorttyp')}</td>                           <td>${c['attributes'][standorttyp] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.vbs.kataster-belasteter-standorte-militaer.status_altlv')}</td>                          <td>${c['attributes'][status_altlv] or '-'}</td></tr>
+    <tr><td class="cell-left" valign="top">${_('ch.vbs.kataster-belasteter-standorte-militaer.untersuchungsmassnahmen')}</td>  <td>${_(str_output)|n}</td></tr> 
+    <tr><td class="cell-left">${_('ch.vbs.kataster-belasteter-standorte-militaer.url_kbs_auszug')}</td>                        <td><a href="${c['attributes']['url_kbs_auszug'] or '-'}" target="_blank">${_('link')}</a></td></tr>
+    <tr><td>&nbsp;</td>                                                                                                       <td>&nbsp;</td></tr>
```

Comment:
what does the last line serve for? &nbsp;?

Reply:
I don't know so i removed it.